CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ Copy and pasting the git commit messages is __NOT__ enough.
 - Allow specifying "-latest" as a version suffix for zoo locator strings.
 - Extended Imitation Learning codebase to allow importing traffic histories from the Waymo motion dataset and replay in a SMARTS simulation. See PR #1060.
 - Added `ros` extension rule to `setup.py`.
+- Added `ParallelEnv` class to simulate multiple SMARTS environments in parallel, with synchronous or asynchronous episodes.
```

Comment:
I think you should also mention `CloudpickleWrapper` and the added parallel environment example.

Reply:
Changelog has been changed accordingly.